### DISCUSSION TITLE
New to Homebridge, getting an error on setup

### DISCUSSION THREAD

**Discussion by evanaczon** [maxileith/homebridge-appletv-enhanced#826] [Q&A] [answered]
Hey all, very new to Homebridge (and Github + Python, tbh) but hoping to get some help. Tried to follow the setup guide but am getting stuck. I've updated everything to be current and am still getting a `Process Ended. Code: 1, Signal: null.`

`Unable to execute request B: /Users/evanaczon/.homebridge/appletv-enhanced/.venv/lib/python3.9/site-packages/urllib3/__init__.py:35: NotOpenSSLWarning: urllib3 v2 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 2.8.3'. See: https://github.com/urllib3/urllib3/issues/3020`

That link above has some stuff about LibreSSL + urllib3 and I don't know anything about that. Is there anything else I can do?

Currently running:
- Mac Mini - Sequoia 15.1.1
- Apple TV HD - tvOS 18.1
- Node.js - 22.12.0
- OpenSSL 3.3.6
- Python 3.9.6

**Comment by maxileith** [comment, marked as answer]
See https://github.com/maxileith/homebridge-appletv-enhanced/issues/62
